packages/devextreme-vue/src/core/component.ts

```diff
@@ -296,6 +297,14 @@ function cleanWidgetNode(node: Node) {
   return removedNodes;
 }
 
+function pickOutDxClasses(el: Element, attr: any) {
+  el.classList.forEach((item: string) => {
+    if (attr.class && attr.class.indexOf(item) === -1) {
+      dxClasses.push(item);
```

Comment:
Is it okay that we only accumulate class names in this array and never remove them? Could there be duplicates?